pkg/providers/vsphere/vmlifecycle/create_contentlibrary.go

```diff
@@ -60,6 +62,36 @@ func deployOVF(
 	return vcenter.NewManager(restClient).DeployLibraryItem(vmCtx, item.ID, deploy)
 }
 
+func deployISO(
+	vmCtx pkgctx.VirtualMachineContext,
+	vimClient *vim25.Client,
+	item *library.Item,
+	createArgs *CreateArgs) (*vimtypes.ManagedObjectReference, error) {
+
+	vmCtx.Logger.Info("Deploying ISO Library Item", "itemID", item.ID, "itemName", item.Name)
+	createConfigSpec := createArgs.ConfigSpec
+	createConfigSpec.GuestId = vmCtx.VM.Spec.GuestID
+
+	vmFolder := object.NewFolder(vimClient, vimtypes.ManagedObjectReference{Type: "Folder", Value: createArgs.FolderMoID})
+	resourcePool := object.NewResourcePool(vimClient, vimtypes.ManagedObjectReference{Type: "ResourcePool", Value: createArgs.ResourcePoolMoID})
+
+	vmCtx.Logger.Info("Creating VM with params", "vmFolder", vmFolder.Reference(), "resourcePool", resourcePool.Reference(), "createConfigSpec", createConfigSpec)
+	task, err := vmFolder.CreateVM(vmCtx, createConfigSpec, resourcePool, nil)
+	if err != nil {
+		vmCtx.Logger.Error(err, "Failed to create VM")
+		return nil, err
+	}
+	taskInfo, err := task.WaitForResultEx(vmCtx)
+	if err != nil {
+		vmCtx.Logger.Error(err, "Task failed to create VM")
+		return nil, err
+	}
+
+	vmCtx.Logger.Info("Successfully Created the VM", "taskInfo", taskInfo)
```

Comment:
I think this was from my PoC for debugging purposes. We probably don't need to print out the entire `taskInfo` for every VM deployment.